src/components/RightPanel.tsx

```diff
@@ -7,21 +7,22 @@ import Trends from "../containers/Trends";
 
 const Wrapper = styled.div``;
```

Comment:
remove this styled div, use default

Reply:
when I remove it. the right panel covers the left panel

Reply:
I don't understand how Wrapper(without styles) !== div.  Maybe you're just deleting this Wrapper?

Reply:
When I delete Wrapper the right panel slides to the left